server/game/cards/01-Core/FearsomeMystic.js

```diff
@@ -9,7 +9,7 @@ class FearsomeMystic extends DrawCard {
 
         this.action({
             title: 'Remove fate from characters',
-            condition: context => context.source.isParticipating(),
+            condition: context => context.source.isParticipating() && context.player.opponent,
             gameAction: ability.actions.removeFate(context => ({
                 target: this.game.currentConflict.getCharacters(context.player.opponent).filter(card => card.getGlory() < context.source.getGlory())
             }))
```

Comment:
No this seems over kill. Better to return an empty array when the player passed to getCharacters doesn't exist